scapy/arch/linux.py

```diff
@@ -621,16 +606,15 @@ def recv(self, x=MTU):
                     "family=%i). Using %s", sa_ll[0], sa_ll[1], sa_ll[3],
                     cls.name)
 
-        try:
-            pkt = cls(pkt)
-        except KeyboardInterrupt:
-            raise
-        except:
-            if conf.debug_dissector:
-                raise
-            pkt = conf.raw_layer(pkt)
-        pkt.time = get_last_packet_timestamp(self.ins)
-        pkt.direction = sa_ll[2]
+        ts = get_last_packet_timestamp(self.ins)
+        # direction = sa_ll[2]
+        return cls, pkt, ts  # , direction
+
+    def recv(self, x=MTU):
```

Comment:
Are these comments needed?

Reply:
I have no idea what the "direction" represents, so as the PR removes this feature (which is only available and used in this function), I left the comments.

Otherwise, we could simply remove this "recv" function, as it only calls its super one.

Reply:
I means that the packet was sent locally, see http://man7.org/linux/man-pages/man7/packet.7.html We don't use at al.